scipy/stats/tests/test_multivariate.py

```diff
@@ -2085,6 +2093,21 @@ def test_haar(self):
         ks_tests = [ks_2samp(proj[p0], proj[p1])[1] for (p0, p1) in pairs]
         assert_array_less([ks_prob]*len(pairs), ks_tests)
 
+    def test_one_by_one(self):
+        # Test that the 1x1 distribution gives ±1 with equal probability.
+        dim = 1
+        xs = ortho_group.rvs(dim, size=5000)
+        np.random.seed(514)
```

Comment:
We're in the process of phasing out use of `np.random.seed` throughout the codebase. Please use `rng = np.random.default_rng(4825489765835639)` and pass `rng` to `rvs`.

Reply:
I switched to division by determinant. This should result in compensating the error from `ortho_group`, if that error occurs.